templates/pvc.yaml

```diff
@@ -31,13 +33,29 @@ kind: PersistentVolumeClaim
 metadata:
   name: dispatcher-filelogdir
 spec:
-  storageClassName: {{ .Values.storage_class_rwmany }} 
+  storageClassName: {{ .Values.storage_class }} 
   accessModes:
-    - ReadWriteMany
+    - ReadWriteOnce
+      #- ReadWriteMany
   resources:
     requests:
       storage: 30Gi
 ---
 # PersistentVolumeClaims related to dda and ddcache should be defined only in dda, since they are useless in the dispatcher if dda is not deployed
 
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: data-reduced-ddcache
+spec:
+  storageClassName: {{ .Values.storage_class }} 
+  accessModes:
+    - ReadWriteOnce
+      #- ReadWriteMany
+  resources:
+    requests:
+      storage: 30Gi
+---
+
```

Comment:
But it is created in dda-chart. We intentionally removed it from here previously

Reply:
No more, I changed it, it's necessary if dispatcher is distant from dda. Also, our k8s provider best storageclass does not let ReadWriteMany, so I changed now to this.  

Does it break something for you?

Reply:
OK, then I propose to add a switch here. For the installations without integral, this volume is not necessary at all.

Reply:
I use already existing `mount_dda` switch